src/app/shared/pagination/paginationChangeEvent.interface.ts

```diff
@@ -0,0 +1,7 @@
+import {PaginationComponentOptions} from './pagination-component-options.model';
+import {SortOptions} from '../../core/cache/models/sort-options.model';
+
+export interface PaginationChangeEvent {
```

Comment:
Tiny thing, please add TypeDocs to describe this new interface.  We require TypeDocs/comments for any public classes/interfaces/methods.